packages/mirage-dns/mirage-dns.3.1.2/opam

```diff
@@ -20,7 +20,7 @@ for the [MirageOS unikernel framework](https://mirage.io).
 depends: [
   "ocaml" {>= "4.05.0"}
   "dune"
-  "dns-lwt"
+  "dns-lwt" {>="1.1.2"}
```

Comment:
What's the rational for the changes? this seems to cause a large breakage in the Mirage ecosystem as a lot of package cannot be co-installed anymore.

Reply:
twas a mistake...